includes/data-port/models/class-sensei-import-model.php

```diff
@@ -118,13 +125,60 @@ private function restore_from_source_array( $data ) {
 			if ( null !== $value ) {
 				switch ( $config['type'] ) {
 					case 'int':
-						$value = '' === $value ? null : intval( $value );
+						if ( '' === $value ) {
+							$value = null;
+						} else {
+							if ( ! is_numeric( $value ) || floor( $value ) !== floatval( $value ) ) {
+								$this->add_line_warning(
+									sprintf(
+										// translators: Placeholder is the column name.
+										__( 'Error in column "%s": It must be a whole number.', 'sensei-lms' ),
+										$key
+									),
+									[
+										'code' => 'sensei_data_port_int_sanitation',
```

Comment:
I think we mean `sanitization` here? (Also in two places below)

Reply:
Oops! My bad! https://github.com/Automattic/sensei/pull/3443/commits/55dac7d895da3d5acc1d5c1f2384a0d94292849d